modules/@apostrophecms/area/ui/apos/components/AposWidgetControls.vue

```diff
@@ -223,7 +221,9 @@ export default {
       if (modal) {
         const result = await apos.modal.execute(modal, {
           widget: this.modelValue,
-          field: this.areaField
+          widgetSchema: apos.modules[
```

Comment:
this piece of code was odd before, if we have `action` we emit action and then open the modal. Feels like a `return;` is missing. And depending on what modal is, what are the expected `props` we must set. I don't know where `widgetSchema` is expected, but it's not `AposImageRelationshipEditor`.